src/util/CancellationHandle.h

```diff
@@ -139,9 +142,6 @@ class CancellationHandle {
                 << std::invoke(detailSupplier, AD_FWD(argTypes)...)
```

Comment:
We can also use this PR to make the message clearer and add the check interval.
So that message becomes more like 
"The time since the last timeout check is at least 283 ms, should be at most 50ms"
(Where the 50ms are included in the 283ms).